server/routes/place/html.py

```diff
@@ -66,6 +67,49 @@
     'Continent',
 }
 
+# Location of manually written templates for SEO experimentation in GCS bucket
+SEO_EXPERIMENT_HTML_GCS_DIR = "seo_experiments/active"
+
+# Location of manually written templates for SEO experimentation on local disk
+SEO_EXPERIMENT_HTML_LOCAL_DIR = "config/seo_experiments/html_templates/active/"
+
+# Map DCID of SEO experiment places to their template filename
+SEO_EXPERIMENT_DCID_TO_HTML = {"country/EGY": "Egypt.html"}
+
+
+def get_seo_experiment_template(dcid: str) -> str:
+  """Load page template for SEO experiments
+  
+  Will load files from GCS unless server is running locally, in which case
+  will load from server/config/seo_experiments/html_templates/active instead.
+
+  If a file cannot be found or read, will return empty string instead.
+  """
+  experiment_template = ""
+  try:
+    filename = SEO_EXPERIMENT_DCID_TO_HTML[dcid]
+
+    if current_app.config['LOCAL']:
+      # Load template from local path
+      template_filepath = os.path.join(current_app.root_path,
+                                       SEO_EXPERIMENT_HTML_LOCAL_DIR, filename)
+    else:
+      # Load template from GCS
+      template_filepath = os.path.join("/tmp/seo_experiment/template/",
+                                       filename)
+      gcs.download_blob("datcom-website-config",
```

Comment:
Nit: extract bucket to a constant

Reply:
Instead of downloading the template to a temporary directory, why not read directly from GCS to a string in memory?

Reply:
Good idea. Extracted the bucket name to a constant (reusing already available constant), and added a new function to read directly from GCS to a text string.